andela/ah-the-unsullied-frontend
# #162949165 Setup testing environment

#### What does this PR do?

Setup testing tools jest and enzyme.

#### Description of Task to be completed?
- [x] Install jest, enzyme, and enzyme-adapter-react-16
- [x] Test components

#### How should this be manually tested?

* Git clone this repo: `git clone https://github.com/andela/ah-the-unsullied-frontend.git`
* Cd to the cloned repo
* Git checkout to ch-enviroment-testing-162949165

* npm run tests and all tests should pass

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#162949165](https://www.pivotaltracker.com/story/show/162949165)

#### Questions:
